src/components/product-menu-dropdown/__tests__/product-menu-dropdown-test-cases.js

```diff
@@ -0,0 +1,15 @@
+import { ProductMenuDropdown } from '../product-menu-dropdown';
```

Comment:
We should decide whether to favor default or named exports by default in the repo and stick with that. I'm fine with either way.

Reply:
Per chat, we're going to stick with `always named` for now.